cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/batch/MapReduceProgramRunnerTest.java

```diff
@@ -312,9 +314,18 @@ public void testMapReduceDriverResources() throws Exception {
 
   @Test
   public void testMapreduceWithObjectStore() throws Exception {
-    final ApplicationWithPrograms app = deployApp(AppWithMapReduceUsingObjectStore.class);
+    // Deploy apps to another namespace and test cross-namespace access meanwhile
+    final ApplicationWithPrograms app = deployApp(NamespaceId.fromString("namespace:someOtherNameSpace").toId(),
```

Comment:
Because its redundant?
On Jun 30, 2016 4:41 PM, "Terence Yim" <notifications@github.com> wrote:

> In
> cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/batch/MapReduceProgramRunnerTest.java
> <https://github.com/caskdata/cdap/pull/5958#discussion_r69228339>:
>
> > @@ -312,9 +314,18 @@ public void testMapReduceDriverResources() throws Exception {
> >
> >    @Test
> >    public void testMapreduceWithObjectStore() throws Exception {
> > -    final ApplicationWithPrograms app = deployApp(AppWithMapReduceUsingObjectStore.class);
> > +    // Deploy apps to another namespace and test cross-namespace access meanwhile
> > +    final ApplicationWithPrograms app = deployApp(NamespaceId.fromString("namespace:someOtherNameSpace").toId(),
>
> @anwar6953 <https://github.com/anwar6953> I had some conversion with
> @bdmogal <https://github.com/bdmogal> before that the Ids class should
> get removed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/caskdata/cdap/pull/5958/files/71653c4d52391d1a8f69c5ae55705417dcf56b5b#r69228339>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACU_EcJ1I3qmTtacn81lU2NliSEuveFdks5qRFQmgaJpZM4JCnzq>
> .
>